python/core/auto_generated/qgsvectorfilewriter.sip.in

```diff
@@ -247,7 +247,8 @@ Creates a clone of the FieldValueConverter.
         bool forceMulti = false,
         bool includeZ = false,
         const QgsAttributeList &attributes = QgsAttributeList(),
-        QgsVectorFileWriter::FieldValueConverter *fieldValueConverter = 0
+        QgsVectorFileWriter::FieldValueConverter *fieldValueConverter = 0,
```

Comment:
These changes are an API break - because python code will get an extra return value and complain that there's too many values to unpack. You'll need to hide the extra argument from SIP for now.

Reply:
@nyalldawson , is there a way to hide individual arguments from SIP, or do we have to maintain separate functions and use SIP_SKIP?